initializr-service/src/main/resources/application.yml

```diff
@@ -1011,28 +1017,24 @@ initializr:
       content:
         - name: Cloud Bus
           id: cloud-bus
-          aliases:
-            - cloud-bus-amqp
-          description: A simple control bus defaulting to RabbitMQ
+          description: A simple control bus using Spring Cloud Stream (requires a binder, e.g. Kafka or RabbitMQ)
           groupId: org.springframework.cloud
-          artifactId: spring-cloud-starter-bus-amqp
+          artifactId: spring-cloud-bus
         - name: Cloud Stream
           id: cloud-stream
-          aliases:
-            - cloud-stream-binder-rabbit
-          description: Messaging microservices with Spring Cloud Stream and RabbitMQ
+          description: Messaging microservices with Spring Cloud Stream (requires a binder, e.g. Kafka or RabbitMQ)
           versionRange: 1.3.0.RELEASE
           groupId: org.springframework.cloud
-          artifactId: spring-cloud-starter-stream-rabbit
+          artifactId: spring-cloud-stream
```

Comment:
So we're at the core of my question here. What happens if I had `spring-kafka` here? Do I get _exactly_ the same behaviour as adding the previous kafka binder? 

(I know the answer to my own question).

If the concept does not exist in the project, start.spring.io should not pretend it exists. If that's what we want, it needs to be a first class concept in Spring Cloud Stream.

Reply:
I thought I was just copying your logic (removing the core jar and adding a starter). I'm quite happy to leave it in (so core + kafka). It's just like the GCP SQL use case (the starters end up not being used by Initializr).